### PR TITLE
🛠 WIP - DCOS-12802: Introduce Jenkinsfile

### PR DESCRIPTION
**Pull request by orlandohohmeier**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@ic/dcos-ui-flavor') _
+
+def branches = ["orlandohohmeier/feat/DCOS-12802-introduce-jenkinsfile",] as String[]
```
Comment (nLight) [inline Jenkinsfile]: So we gonna change it afterwards, correct?

Comment (orlandohohmeier) [inline Jenkinsfile]: Yep... 😆

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      steps {
+        echo "Build ID: ${env.BUILD_ID}"
+        echo "Chnage ID: ${env.CHANGE_ID}"
+        echo "Change Author: ${env.CHANGE_AUTHOR}"
+      }
+    }
+
+    stage("Verify Author") {
+      when {
+        expression { return env.CHANGE_AUTHOR != "orlandohohmeier" }
```
Comment (nLight) [inline Jenkinsfile]: What if we store authors in Jenkins and inject the value in the script?

**Comment by orlandohohmeier** [comment]
Closed this PR to run a few experiments...

**Pull request closed without merging**